tests/Features/Depends.php

```diff
@@ -0,0 +1,40 @@
+<?php
```

Comment:
Wondering if we can add 2 two tests more:

The first is incomplete, the seconds depends of the first. Should it shouldn't run. Do you know what I mean?

Reply:
Like in the phpunit examples

Empty returns empty array
Push receives empty array adds item and returns array
Assume receives array and assets?

Reply:
```php
test('incomplete')->skip('incomplete');
test('depends on incomplete')->depends('incomplete');
```
works fine
![image](https://user-images.githubusercontent.com/13331388/85176574-65afdd80-b27a-11ea-9d5d-892df0bea97c.png)

The message could be more specific

Reply:
@nuernbergerA I think is fine for now.

Reply:
Should I add them as they are?
Or add a test that expects this result?

Reply:
Add this on the `tests/Features/Depends.php`.

```
test('incomplete')->skip('incomplete');
test('depends on incomplete')->depends('incomplete');
```

Reply:
Make sure you run `composer update:snapshots`.

Reply:
@nunomaduro this will lead to problems on `composer test` and exits with code 1

![image](https://user-images.githubusercontent.com/13331388/85180098-dd820600-b282-11ea-8e12-3bc7b844abf0.png)

`composer test:update:snapshots`
![image](https://user-images.githubusercontent.com/13331388/85180194-13bf8580-b283-11ea-9b9d-f084298d88ad.png)

`composer test`
![image](https://user-images.githubusercontent.com/13331388/85180235-38b3f880-b283-11ea-8ea0-01f0d2e115fe.png)

Reply:
Add those changes, I will fix snapshots in my side.

Reply:
@nunomaduro done